Formula/mpv.rb

```diff
@@ -0,0 +1,56 @@
+class Mpv < Formula
+  desc "Media player based on MPlayer and mplayer2"
+  homepage "https://mpv.io"
+  url "https://github.com/mpv-player/mpv/archive/v0.30.0.tar.gz"
+  sha256 "33a1bcb7e74ff17f070e754c15c52228cf44f2cefbfd8f34886ae81df214ca35"
+  head "https://github.com/mpv-player/mpv.git"
+
+  depends_on "docutils" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python" => :build
+
+  depends_on "ffmpeg"
+  depends_on "jpeg"
+  depends_on "libarchive"
+  depends_on "libass"
+  depends_on "little-cms2"
+  depends_on "lua@5.1"
+
+  depends_on "mujs"
+  depends_on "uchardet"
+  depends_on "vapoursynth"
+  depends_on "youtube-dl"
+
+  def install
+    # LANG is unset by default on macOS and causes issues when calling getlocale
+    # or getdefaultlocale in docutils. Force the default c/posix locale since
+    # that's good enough for building the manpage.
+    ENV["LC_ALL"] = "C"
+
+    args = %W[
+      --prefix=#{prefix}
+      --enable-html-build
+      --enable-javascript
+      --enable-libmpv-shared
+      --enable-lua
+      --enable-libarchive
+      --enable-uchardet
+      --confdir=#{etc}/mpv
+      --datadir=#{pkgshare}
+      --mandir=#{man}
+      --docdir=#{doc}
+      --zshdir=#{zsh_completion}
+    ]
+
+    system "./bootstrap.py"
+    system "python3", "waf", "configure", *args
+    system "python3", "waf", "install"
+
+    system "python3", "TOOLS/osxbundle.py", "build/mpv"
+    prefix.install "build/mpv.app"
```

Comment:
Don't install a `.app` for Homebrew/homebrew-core, thanks. That's what `homebrew/cask` is for.

Reply:
thank @MikeMcQuaid , it seems that install as a `.app` was an option, then it became default behavior in this commit https://github.com/Homebrew/homebrew-core/commit/1fdd9b152760e9d743964dd955c1ae816a3ca191  do you want me to add the `with-bundle` option back?

Reply:
No option, simply do not install the app. People who want the app can install from cask

Reply:
deleted in https://github.com/Homebrew/homebrew-core/pull/45854/commits/0c7e9af5fd5b41024dba277a5eef59c974168fa5